bootstrap.php

```diff
@@ -43,9 +42,10 @@
 // Load Composer autoloader:
 require_once(dirname(__FILE__) . '/vendor/autoload.php');
 
+\PHPCI\ErrorHandler::register();
+
 if (defined('PHPCI_IS_CONSOLE') && PHPCI_IS_CONSOLE) {
     $loggerConfig = LoggerConfig::newFromFile(__DIR__ . "/loggerconfig.php");
```

Comment:
I think for now we can do option 1 and maybe in the future option 3. (Option 2 doesn't sound good for me)